src/Microsoft.InsightsGenerator/DataTransformation.cs

```diff
@@ -34,6 +34,7 @@ public DataArray Transform(DataArray array)
 
         private string[] GetColumnLabels(DataArray array, out DataArray.DataType[] columnDataType)
         {
+            #pragma warning disable CA1854
```

Comment:
comments explaining why?

Reply:
sure, I can a comment.  The SDK update is bringing in a bunch of additional static analysis, that I'd rather fix incrementally than as part of the bump.  I'll also create an issue to track following up on reenabling the disabled rules.